geotrek/maintenance/tests/test_views.py

```diff
@@ -651,14 +651,14 @@ def setUpTestData(cls):
 
         cls.job1 = InterventionJobFactory(job="Worker", cost=12)
         cls.job2 = InterventionJobFactory(job="Streamer", cost=60)
-        cls.job1_column_name = "Cost Worker"
-        cls.job2_column_name = "Cost Streamer"
+        cls.job1_column_name = "Cost_Worker"
+        cls.job2_column_name = "Cost_Streamer"
```

Comment:
Why adding an underscore ?

Reply:
I see, thank you for the explanation :)

Reply:
Django 3.2.13 : https://docs.djangoproject.com/en/3.2/releases/3.2.13/